cli.js

```diff
@@ -41,5 +45,7 @@ if (process.stdin.isTTY) {
 
 	init(cli.input);
 } else {
-	process.stdin.pipe(isProgressive.stream(init));
+	isProgressive.stream(process.stdin).then(p => {
+		log(p, 'file');
+	});
```

Comment:
It's not possible to retrieve the filename of stdin, stdin might not even come from a file.